assets/Gruntfile.js

```diff
@@ -53,6 +53,14 @@ module.exports = function (grunt) {
         files: '<%= yeoman.app %>/styles/*.less',
         tasks: ['less']
       },
+      extensionJS: {
+        files: ['extensions/js/{,*/}*.js'],
```

Comment:
maybe include a dummy "extensions/js/example.js" file containing only a comment, and eventually a link to doc for how to include an external js file in the real console?